misc/docerina/src/main/java/org/ballerinalang/docgen/generator/model/Type.java

```diff
@@ -202,15 +202,17 @@ public static Type fromNode(Node node, SemanticModel semanticModel) {
         } else if (node instanceof FunctionTypeDescriptorNode) {
             type.isLambda = true;
             FunctionTypeDescriptorNode functionDescNode = (FunctionTypeDescriptorNode) node;
-            FunctionSignatureNode functionSignature = functionDescNode.functionSignature();
-            List<DefaultableVariable> variables =
-                    Generator.getDefaultableVariableList(functionSignature.parameters(), Optional.empty(),
-                            semanticModel);
-            type.paramTypes.addAll(variables.stream().map((defaultableVariable) -> defaultableVariable.type)
-                    .collect(Collectors.toList()));
-            if (functionSignature.returnTypeDesc().isPresent()) {
-                ReturnTypeDescriptorNode returnType = functionSignature.returnTypeDesc().get();
-                type.returnType = Type.fromNode(returnType.type(), semanticModel);
+            if (functionDescNode.functionSignature().isPresent()) {
```

Comment:
We'll probably need doc generation support for `function` too, right? Can you please create an issue?

Reply:
Created. (https://github.com/ballerina-platform/ballerina-lang/issues/28557)